misk-hibernate-testing/src/main/kotlin/misk/jdbc/TruncateTablesService.kt

```diff
@@ -23,13 +23,15 @@ private val logger = getLogger<TruncateTablesService>()
  */
 class TruncateTablesService(
   private val qualifier: KClass<out Annotation>,
-  private val config: DataSourceConfig,
+  internal val connector: DataSourceConnector,
   private val transacterProvider: Provider<Transacter>,
   private val startUpStatements: List<String> = listOf(),
   private val shutDownStatements: List<String> = listOf()
 ) : AbstractIdleService() {
   private val persistentTables = setOf("schema_version")
 
+  private val config get() = connector.config()
```

Comment:
I’d avoid this because it doesn’t act like a field. (you get a different value if you access it before startup, right?)

Reply:
I suppose that `private val config = connection.config()` is preferred then? Will fix.